### PR TITLE
polybar: update to 3.5.0.

### PR DESCRIPTION
**Pull request by pbui**

### CODE REVIEW

**Comment by pbui** [comment]
Yeah, I just saw that... What is the best way to handle that?

**Comment by ericonr** [comment]
Check for `XBPS_TARGET_NO_ATOMIC8` and add `makedepends+=" libatomic-devel"`. If you're lucky it just works; otherwise you'll need to add `LDFLAGS` or `LDLIBS` that are `=-latomic`.

**Comment by pbui** [comment]
Ok, I added the same workaround/fix as was used in `musikcube`.

File: srcpkgs/polybar/template [hunk 1]
```diff
@@ -46,6 +46,11 @@ if [ "$build_option_network" ]; then
 	esac
 fi
 
+if [ "$XBPS_TARGET_NO_ATOMIC8" ]; then
+	makedepends+=" libatomic-devel"
+	configure_args+=" -DCMAKE_EXE_LINKER_FLAGS='-latomic'"
```
Comment (ericonr) [inline srcpkgs/polybar/template:51]: Linker flags aren't the same as libraries the binary is linked against. If it works, I'd prefer to use `STANDARD_LIBRARIES`.

Comment (pbui) [reply srcpkgs/polybar/template]: Ok, I've pushed an update.

**Pull request merged**